.changeset/mighty-cycles-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/eip-1193-provider": patch
+---
+
+Tweak unit tests given expectation that the wallet ID used for tests will have 2 associated addresses
```

Comment:
Nit: if it's a test-only change and doesn't impact the code that our customers consume whatsoever there's no need for a changelog entry 💅

Reply:
sahweet! removed :)